tests/area/test_markets.py

```diff
@@ -49,15 +50,18 @@ def area_fixture(self):
         ConstSettings.BalancingSettings.ENABLE_BALANCING_MARKET = False
         ConstSettings.SettlementMarketSettings.ENABLE_SETTLEMENT_MARKETS = False
         gsy_e.constants.RETAIN_PAST_MARKET_STRATEGIES_STATE = False
+        original_future_markets = GlobalConfig.FUTURE_MARKET_DURATION_HOURS
```

Comment:
Caching the previous value of the setting and restoring it right after the test has been completed.